libraries/AP_GPS/AP_GPS.h

```diff
@@ -176,6 +176,8 @@ class AP_GPS
         return status(primary_instance);
     }
 
+    bool is_healthy(bool record_log, bool send_feedback);
```

Comment:
Very well i thought about it and on second thought it doesnt sound so good. Will correct it.